src/components/ModalConfirmCode.tsx

```diff
@@ -0,0 +1,104 @@
+import { connected } from 'process';
+import React,  {FC, useEffect, useState} from 'react';
+import './ModalConfirmCode.scss';
+
+interface CardProps {
+  active: boolean;
+  setActive: (bool: boolean) => void;
+}
+
+const ModalConfirmCode: FC<CardProps> = 
+  ({
+    active, 
+    setActive,
+  }) => {
+  
+  const [code, setCode] = useState('');
+  const [codeDirty, setCodeDirty] = useState(false);
```

Comment:
codeIsDirty, setCodeIsDirty - use 'is' or 'has' for naming of boolean variables

Reply:
done